bench/pathway.js

```diff
@@ -0,0 +1,21 @@
+var Graph  = require('../dist/src/Graph')
+var time   = require('microtime')
+
+var SIZE  = 1000000
+var graph = new Graph()
+
+console.log('Creating a single pathway at %s length', SIZE)
+
+var start = time.now()
+while (SIZE--) {
+  graph.append(SIZE)
+}
+var after = time.now()
+
+console.log('Appending:', (after - start) / 1000 + 'ms')
+
+var beginPath = time.now()
+var items = graph.pathway()
+var endPath = time.now()
+
+console.log('Pathway:', (endPath - beginPath) / 1000 + 'ms')
```

Comment:
Hopefully we will never have a single path at 1 million nodes. The benchmark at ten thousand nodes is still impressive:

```
●° node bench/pathway.js
Creating a single path at 10000 length
Appending: 2.67ms
Pathway: 1.315ms
```